subsys/logging/backends/log_backend_net.c

```diff
@@ -196,20 +197,31 @@ static int format_set(const struct log_backend *const backend, uint32_t log_type
 	return 0;
 }
 
+bool log_backend_net_set_addr(const char *addr)
```

Comment:
Please fix: Shouldn't this function's definition be excluded without `LOG_BACKEND_NET_SERVER_RUNTIME`?

Reply:
Is that comment double or triple negative?

Banter aside, probably not since it's used no matter what. As you can see it is called from `init_net()` when `LOG_BACKEND_NET_SERVER_RUNTIME` is _not_ defined (aka when `LOG_BACKEND_NET_SERVER_STATIC` _is_ defined)

Reply:
Ugh, sorry for the brain-dead reading.

Reply:
No worries it happens to all of us